src/index.ts

```diff
@@ -11,27 +11,27 @@ const DEFAULTS = {
   BASE_URL: "https://api.courier.com"
 };
 
+const getEnvVariable = (name: string) => globalThis?.process?.env?.[name];
```

Comment:
do we need to change this with this PR? Unless its absolutely necessary I'd suggest keeping it isolated from axios -> fetch changes?

Reply:
This is necessary to get it to run on Cloudflare workers as there's the `process` global variable there.